finality-aleph/src/sync/handler.rs

```diff
@@ -19,28 +21,35 @@ use crate::{
 const MAX_JUSTIFICATION_BATCH: usize = 100;
 
 /// Handler for data incoming from the network.
-pub struct Handler<I: PeerId, J: Justification, CS: ChainStatus<J>, V: Verifier<J>, F: Finalizer<J>>
-{
+pub struct Handler<
+    B: Block,
+    I: PeerId,
+    J: Justification,
+    CS: ChainStatus<J>,
+    V: Verifier<J>,
+    F: Finalizer<J>,
```

Comment:
At this point it's probably better to use `where`. <_<"

Reply:
Here all of them have additional constraints, but maybe we should startnaming those params more descriptively.

Reply:
fixed
@fixxxedpoint - welp, this would require a separate task, if not an epic...